### DISCUSSION TITLE
Latest version

### DISCUSSION THREAD

**Discussion by real-dialga** [termux/termux-app#2345] [Q&A] [answered]
From where can I install latest version of termux??

**Comment by agnostic-apollo** [comment, marked as answer]
Go to https://github.com/termux/termux-app#installation and read.

**Reply by leogama** [reply]
Where are versions v0.118 and above cited in the README? They were not released yet?

**Reply by agnostic-apollo** [reply]
Latest is `v0.117`. Docs are updated before releases.

**Reply by leogama** [reply]
A little counterintuitive, but OK.

**Reply by agnostic-apollo** [reply]
Not counterintuitive. Can't go back in time or git history after a release has already been made for updating docs with info of what features the new release will bring. Version updates, changelogs, docs, etc are all updated before releases. Nonetheless, I do plan on adding currently latest version info to `README`.

**Comment by Redeem01** [comment]
Or download from F-droid
